addons/portal/static/src/js/portal_chatter.js

```diff
@@ -254,6 +257,26 @@ var PortalChatter = publicWidget.Widget.extend({
         var page = $(ev.currentTarget).data('page');
         this._changeCurrentPage(page);
     },
+
+    /**
+     * @private
```

Comment:
Also add `* @param {Event} ev`

Reply:
Remembering old cr, uid, ids, context unnecessary dozen of comment lines in old openerp.